src/dotnet-serve/Startup.cs

```diff
@@ -62,6 +67,15 @@ public void ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app)
         {
+            if (_options.EnableCors)
+            {
+                app.UseCors(corsPolicy =>
```

Comment:
Would it help us debug failures if this came after statuscodepages and developerexception middleware?

Reply:
Tried that, still failing. I created Github Action on my fork, this is what I am getting in exception of the test case which I have written to test OPTIONS request.
![image](https://user-images.githubusercontent.com/1421986/74098330-fb08cb80-4b3c-11ea-850b-f3ae4257b3ce.png)

The test case passed on windows image in my Github Action too. It is failing due to newly added `-c` option, but again it is failing on Linux and Mac and working on Windows 😢.

Reply:
The problem is that your test code isn't retrying long enough. It takes a second or two for dotnet-serve to start accepting requests. I use retries in all other tests. See https://github.com/natemcmaster/dotnet-serve/blob/master/test/dotnet-serve.Tests/HttpClientExtensions.cs

Reply:
Implemented the retrying mechanism, and now all the build is passing.

Reply:
Thanks for fixing! I'll get this released to nuget.org momentarily...